scenarios/monitoring/cdp_monitoring/incremental_ingest.dig

```diff
@@ -48,6 +48,31 @@ schedule:
         _env:
             TD_API_KEY: ${secret:td.apikey}
 
++incremental_ingest_activations:
+    +append_activations_history:
+        td>:
+        query: select * from ${td.tables.activations}
+        insert_into: ${td.tables.activations_history}
+
+    +get_current_parent_segment_list:
+        td>:
+        query: select ARRAY_JOIN(ARRAY_AGG(id), ',') as ids from ${td.tables.parent_segments}
+        store_last_results: true
+        database: ${td.database}
+
+    +ingest_activations:
+        py>: scripts.ingest_activation.run
+        session_unixtime: ${session_unixtime}
+        dest_db: ${td.database}
+        dest_table: ${td.tables.activations}
+        ids: ${td.last_results.ids}
+        api_endpoint: ${td.api_endpoint}
+        cdp_api_endpoint: ${td.cdp_api_endpoint}
+        docker: 
+            image: "digdag/digdag-python:3.9"
```

Comment:
[nitpick] It'd be ok to use `3.10.1` instead because this is a sample and the version is our latest.